indico/modules/events/blueprint.py

```diff
@@ -42,6 +43,10 @@
 _bp.add_url_rule('/event/create/<any(lecture,meeting,conference):event_type>', 'create', RHCreateEvent,
                  methods=('GET', 'POST'))
 
+# Preparation
+_bp.add_url_rule('/event/prepare', 'prepare', RHPrepareEvent,
+                 methods=('GET', 'POST'))
```

Comment:
This fits nicely in one line and we do not need GET

```suggestion
_bp.add_url_rule('/event/prepare', 'prepare', RHPrepareEvent, methods=('POST',))
```

Reply:
Right, thanks.